cypress/support/config/services.js

```diff
@@ -2,6 +2,7 @@ export default {
   igbo: {
     pageTypes: {
       articles: undefined,
+      fontStartsWith: undefined,
```

Comment:
This should be one level up in the config, it shouldn't be under pageTypes

Reply:
Done